Update README, CONTRIBUTING, and FAQ (#6712)
elastic/eui: CONTRIBUTING.md

🙌 Thanks for your interest in contributing to EUI! 🙌

> If there isn't an associated feature request or bug report in EUI's backlog yet, [please create an issue](https://github.com/elastic/eui/issues/new) so that you get a chance to discuss the changes you have in mind with the EUI team so we can scope out your work and provide guidance and recommendations.

Author: ha yup I definitely ran the fuck out of that sentence. Thanks Trevor!
```suggestion
If there isn't an associated feature request or bug report in EUI's backlog yet, [please create an issue](https://github.com/elastic/eui/issues/new) so that you get a chance to discuss the changes you have in mind with the EUI team. This will allow us to help you scope out your work and provide guidance and recommendations.
```
